helpers/compression/compression_test.go

```diff
@@ -0,0 +1,68 @@
+//go:build compress
+// +build compress
```

Comment:
what is this?

Reply:
this can be used for `--tags` when running tests to run just these tests.